ci/docker/linux-apt-docs.dockerfile

```diff
@@ -48,16 +61,20 @@ RUN apt-get update -y && \
         libxml2-dev \
         meson \
         ninja-build \
+        nodejs \
+        npm \
         nvidia-cuda-toolkit \
         openjdk-${jdk}-jdk-headless \
         pandoc \
         r-recommended=${r}* \
         r-base=${r}* \
         rsync \
         ruby-dev \
+        sudo \
         wget && \
     apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    npm install -g yarn @mermaid-js/mermaid-cli
```

Comment:
We should be setting 

```
ENV CHROME_BIN="/usr/bin/chromium" \
    PUPPETEER_SKIP_CHROMIUM_DOWNLOAD="true"
```

before we install mermaid-cli so that puppeteer doesn't download its own chromium binary, changing `CHROME_BIN`'s value to wherever Debian installs it

Reply:
Thanks for the info!
It seems that `CHROME_BIN` is needless. We need `PUPPETEER_EXECUTABLE_PATH` instead.